etc/version.lock.json

```diff
@@ -1648,7 +1648,7 @@
     "version": 4
   },
   "8acb7614-1d92-4359-bfcf-478b6d9de150": {
-    "rule_name": "Suspicious JAR Child Process",
+    "rule_name": "Suspicious JAVA Child Process",
```

Comment:
You can undo this change - this file is only updated programmatically during the lock process

Reply:
```suggestion
    "rule_name": "Suspicious JAR Child Process",
```